### DISCUSSION TITLE
How do I view one page at a time？

### DISCUSSION THREAD

**Discussion by peng-yo** [ahrm/sioyek#582] [Q&A]
because I want to see the presentation

**Comment by ahrm** [comment]
There is a `toggle_presentation_mode` command.

**Reply by peng-yo** [reply]
Thanks! And where is the configuration? mac

**Reply by ahrm** [reply]
What configuration? All configurations files can be placed in `~/.config/sioyek`, but `toggle_presentation_mode` is a command, not a configuration.

**Reply by peng-yo** [reply]
I have some error with my F10 and I want to change his key binding to F1

**Reply by ahrm** [reply]
You can put it in `~/.config/sioyek/keys_user.config`

**Reply by peng-yo** [reply]
Thank you so much
